lsp/Main.java

```diff
@@ -0,0 +1,24 @@
+package lsp;
+
+public class Main {
+
+  /**
+   * Run a demo.
+   * @param args The ususal.
+   */
+  public static void main(String[] args) {
+    Rectangle r = getRectangle();
+    r.setHeight(10);
+    r.setWidth(50);
+    System.out.println(r.area()); // it's not a rectangle!
```

Comment:
Could you also include an explanatory print message in Main? This will benefit the students who miss your explanation in class.

Reply:
Good idea! I added some print statements.